packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -0,0 +1,167 @@
+import { deeplyFulfilledObject, makeTracer } from '@agoric/internal';
+import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+import { E } from '@endo/far';
+import { Stable } from '@agoric/internal/src/tokens.js';
+import { makeGovernedTerms as makeGovernedATerms } from '../auction/params.js';
+import { SECONDS_PER_HOUR, SECONDS_PER_MINUTE } from './econ-behaviors.js';
+
+const trace = makeTracer('NewAuction', true);
+
+/**
+ * @param {import('./econ-behaviors.js').EconomyBootstrapPowers} powers
+ * @param {object} config
+ * @param {any} [config.auctionParams]
+ */
+export const addAuction = async (
+  {
+    consume: {
+      zoe,
+      board,
+      chainTimerService,
+      priceAuthority,
+      chainStorage,
+      economicCommitteeCreatorFacet: electorateCreatorFacet,
+    },
+    produce: { newAuctioneerKit },
+    instance: {
+      consume: { reserve: reserveInstance },
+    },
+    installation: {
+      consume: {
+        auctioneer: auctionInstallation,
```

Comment:
Just to make things easier to understand for me, the reason why `add-auction` uses the existing `auctionInstallation` is to have a new auction vat that uses new instance(s) of `priceFeed` since deployed auctioneer isn't upgradable and does NOT have anything to do with replacing `auctioneer` with its upgradable version. Am I correct? @Chris-Hibbert

Reply:
yes, that's correct.